docs/content/articles/pt/madara-beast-article.md

```diff
@@ -0,0 +1,157 @@
+[Keep Starknet Strange](https://github.com/keep-starknet-strange), é desenhada
+especificamente para perceber a
+[visão fractal scaling](https://medium.com/starkware/fractal-scaling-from-l2-to-l3-7fe238ecfb4f)
+da Starknet. Existem inúmeras razões interessantes para os desenvolvedores
+escolherem em estabelecer uma appchain da Starknet ou L3 ao invés de depender
+diretamente da Starknet.
+
+### Escalabilidade
+
+Desenvolvedores enfrentam desafios significantes em termos de escalabilidade nas
+infraestruturas de blockchain existentes. Escalabilidade abrange dois aspetos
+cruciais: alta velocidade e taxas baixas. Ao implementar uma redução de custo de
+1,000x em cada camada, desenvolvedores conseguem alcançar uma redução de custo
+significativa de L1 até L3, potencialmente alcançando 1,000,000x. A capacidade
+de processamento não é afetada pela atividade de aplicações de terceiros pois a
+aplicação tem uma blockchain dedicada e não compete por recursos. Isto garante
+uma experiência suave e consistente.
+
+### Personalização
+
+Chains de uso geral como Starknet e Ethereum tẽm múltiplas medidas no que toca a
+garantit que a rede é utilizável por todos, levando a um ambiente constragido.
+Com appchains, desenvolvedores podem afinar vários aspetos das suas aplicações e
+infraestruturas, criando soluções à medida. Não gostas de um recurso do Cairo
+VM? Elimina-o da tua appchain.
+
+### Inovação
+
+A personalização de appchains permite aos desenvolvedores trabalhar com recursos
+que estão indisponíveis ou arriscados atualmente em ambientes como o Starknet.
+Appchains oferecem a cada equipa a autonomia para escrever e autorizar quaisquer
+dicas de código desejadas. Isto permite appchains desbloquearem muitos casos de
+uso, como ser capaz de reforçar on-chain KYC sem vazar informação privada.
+
+## Efeito do Madara na Pilha da Appchain
+
+1. **Execução:** A camada de execução define a execução dos blocos e geração de
+   diferença de estado. Madara oferece a flexibilidade para mudar entre dois
+   caixotes de execução,
+   [Blockifier por StarkWare](https://github.com/starkware-libs/blockifier) e
+   [Starknet_in_rust por LambdaClass](https://github.com/lambdaclass/starknet_in_rust).
+   Independente da crate escolhida, a estrutura subjacente utiliza a Cairo VM. A
+   linguagem Cairo facilita a criação de programas prováveis, permitindo a
+   demonstração da execução correta do cálculo.
+2. **Settlement:** Como um settlement rollup, o estado da appchain do Madara
+   pode ser reconstruído exclusivamente ao examinar a sua settlement layer. Ao
+   se estabelecer mais frequentemente na Starknet L2, uma appchain L3 consegue
+   alcançar uma finalidade dura mais rápido, enquanto a descentralização da
+   camada de sequenciamento permite uma finalidade mais robusta e suave. Assim,
+   a settlement é reforçada em ambas as frentes (finalidade dura e suave).
+3. **Sequenciamento:** O Madara assume o controlo do processo de sequenciação,
```

Comment:
u need to be consistent, pick either "sequencer" or "sequenciamento" through the whole document. Probably the pt alternative is better